core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSupervisor.scala

```diff
@@ -50,7 +50,7 @@ class SubmissionSupervisor(executionServiceCluster: ExecutionServiceCluster,
   }
 
   override val supervisorStrategy =
-    OneForOneStrategy(maxNrOfRetries = 3) {
+    OneForOneStrategy(maxNrOfRetries = -1) {
```

Comment:
i guess i could try counting the number of retries and start spamming errors if it gets too high. sentry alert?